src/server.js

```diff
@@ -80,6 +80,16 @@ function allRequests(req, res, next) {
 	return next();
 }
 
+// Information to populate the About section in UI, either from npm or from git
+try {
+	var gitCommit = require("child_process")
+		.execSync("git rev-parse --short HEAD") // Returns hash of current commit
+		.toString()
+		.trim();
+} catch (e) {
+	// Not a git repository or git is not installed: treat it as npm release
```

Comment:
If git is not installed, then we will falsely report that this is an npm release. However, if this is a git repo, chances are git is installed, and is available to the user running the instance.

Alternative is to check if there is a `.git` directory, but this is much more complex (not trivial to find the hash of current commit by just exploring the directory, very error-prone!). It is somewhat not 100% reliable as well as one can have a git repo without a `.git` folder (well, I didn't know), though this case must be super rare. But who knows, maybe some servers centralize `.git` folders, as some do with Python virtualenvs?

Another alternative is to use a JS implementation of git, such as https://github.com/creationix/js-git, but then we're talking über-overkill and another world of issues incoming.

To conclude: not the most reliable, but the simplest solution without a doubt, and because of the `try`/`catch`, it will not break a server for various and random reasons.